.github/workflows/build-history.yml

```diff
@@ -0,0 +1,45 @@
+name: Build results
+
+on:
+  push:
+    branches:
+      - "main"
```

Comment:
Can asv run only missing benchmarks? 

It would be nice to leave this in so when we add a new benchmark it will get back dated for the past x commits at the first push. 

However we don't want to run every benchmark for all these commits on every push?

Reply:
To `asv run` the argument `HEAD -n=x` can be passed to benchmark the last x commits, how many commits should I add?

By using `--skip-existing` it can be made sure that the benchmarks are not run for every push